Babylon/commands/api/connectors/service/api.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import jmespath
+
+from git import Optional
+from pathlib import Path
+from Babylon.utils.environment import Environment
+from Babylon.utils.interactive import confirm_deletion
+from Babylon.utils.request import oauth_request
+from Babylon.utils.response import CommandResponse
+
+logger = logging.getLogger("Babylon")
+env = Environment()
+
+
+class ApiConnectorService:
+
+    def __init__(self, azure_token: str, state: dict = None) -> None:
+        self.state = state
+        self.azure_token = azure_token
+
+    def create(self, connector_file: Path):
+        details = env.fill_template(connector_file)
+        response = oauth_request(
+            f'{self.state["api"]["url"]}/connectors',
```

Comment:
for better readability, could you please create a variable `url = self.state["api"]["url"] `? :slightly_smiling_face: